spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/source/TestSparkReaderDeletes.java

```diff
@@ -167,6 +181,70 @@ public StructLikeSet rowSet(String name, Types.StructType projection, String...
     return set;
   }
 
+  private StructLikeSet actualPositionDeleteRowSet(String tableName, Table table) {
+    return actualPositionDeleteRowSet(tableName, table, null, null);
+  }
+
+  private StructLikeSet actualPositionDeleteRowSet(String tableName, Table table, String filter) {
+    return actualPositionDeleteRowSet(tableName, table, filter, null);
+  }
+
+  private StructLikeSet actualPositionDeleteRowSet(String tableName,
```

Comment:
this and the following have formatting issues I think? We generally don't do 1 arg per line.